packages/requisitions/src/CustomerRequisition/api/hooks.ts

```diff
@@ -1,5 +1,6 @@
 import { useMemo } from 'react';
 import {
+  useHostContext,
```

Comment:
ah.. didn't we speak about this? Was I meant to use the store ID from elsewhere..? Local storage?